lib/collections/env.dart

```diff
@@ -32,23 +32,23 @@ abstract class Env {
     };
   }).toList();
 
-  @EnviedField(varName: 'ENABLE_UPDATE_CHECK', defaultValue: "1")
+  @EnviedField(varName: 'ENABLE_UPDATE_CHECK', defaultValue: 1)
   static final String _enableUpdateChecker = _Env._enableUpdateChecker;
 
   @EnviedField(varName: "RELEASE_CHANNEL", defaultValue: "nightly")
   static final String _releaseChannel = _Env._releaseChannel;
 
-  @EnviedField(varName: "DISABLE_SPOTIFY_IMAGES", defaultValue: "0")
+  @EnviedField(varName: "DISABLE_SPOTIFY_IMAGES", defaultValue: 0)
   static final String _disableSpotifyImages = _Env._disableSpotifyImages;
 
-  static bool get disableSpotifyImages => _disableSpotifyImages == "1";
+  static bool get disableSpotifyImages => _disableSpotifyImages == 1;
 
   static ReleaseChannel get releaseChannel => _releaseChannel == "stable"
       ? ReleaseChannel.stable
       : ReleaseChannel.nightly;
 
   static bool get enableUpdateChecker =>
-      kIsFlatpak || _enableUpdateChecker == "1";
+      kIsFlatpak || _enableUpdateChecker == 1;
```

Comment:
What's the point of this change? Assigning `int` as default value for a `String` field will always cause an exception when the variable is not in the file. Also it's a unrelated change. Please revert it